rq_scheduler/utils.py

```diff
@@ -11,3 +13,9 @@ def from_unix(string):
 def to_unix(dt):
     """Converts a datetime object to unixtime"""
     return calendar.timegm(dt.utctimetuple())
+
+def crontab_get_next_scheduled_time(crontab_string):
+    """Calculate the next scheduled time by creating a crontab object
+    with a crontab string"""
+    job = croniter.croniter(crontab_string, datetime.utcnow())
```

Comment:
`croniter` doesn't return a `Job` instance so I think we should find a more suitable variable name for this.